editor/src/java/com/defold/editor/pipeline/TextureSetGenerator.java

```diff
@@ -312,6 +312,8 @@ public static TextureSetResult calculateLayout(List<Rect> images, List<SpriteGeo
 
         Pair<TextureSet.Builder, List<UVTransform>> vertexData = genVertexData(layout.getWidth(), layout.getHeight(), rects, iterator);
 
+        vertexData.left.setUseGeometries(use_geometries);
```

Comment:
In order to make the files identical, please put the line on the same line as in ./com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/textureset/TextureSetGenerator.java

Reply:
I've removed all differences past the 
```
// The code below must remain identical to the implementation in the editor!
// ./editor/src/java/com/defold/editor/pipeline/TextureSetGenerator.java
```
line using a diff tool. :) Thanks!